src/Cake.Prca.Issues.InspectCode.Tests/packages.config

```diff
@@ -0,0 +1,14 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<packages>
+  <package id="Cake.Core" version="0.18.0" targetFramework="net452" />
+  <package id="Cake.Prca" version="0.1.0-unstable0011" targetFramework="net452" />
+  <package id="Cake.Testing" version="0.18.0" targetFramework="net452" />
+  <package id="Shouldly" version="2.8.2" targetFramework="net452" />
+  <package id="xunit" version="2.2.0" targetFramework="net452" />
+  <package id="xunit.abstractions" version="2.0.1" targetFramework="net452" />
+  <package id="xunit.assert" version="2.2.0" targetFramework="net452" />
+  <package id="xunit.core" version="2.2.0" targetFramework="net452" />
+  <package id="xunit.extensibility.core" version="2.2.0" targetFramework="net452" />
+  <package id="xunit.extensibility.execution" version="2.2.0" targetFramework="net452" />
+  <package id="xunit.runner.visualstudio" version="2.2.0" targetFramework="net452" developmentDependency="true" />
+</packages>
```

Comment:
Already added https://github.com/cake-contrib/Cake.Prca.Issues.InspectCode/issues/2 for this